packages/nuxt/src/app/components/nuxt-link.ts

```diff
@@ -81,12 +81,12 @@ export function defineNuxtLink (options: NuxtLinkOptions) {
     props: {
       // Routing
       to: {
-        type: [String, Object] as PropType<string | RouteLocationRaw>,
+        type: [String, Object] as PropType<RouteLocationRaw>,
         default: undefined,
         required: false
```

Comment:
BTW are these intentional? It should be the exact same without them

Reply:
Let's get rid of them then 😆